factool/utils/openai_wrapper.py

```diff
@@ -27,6 +28,7 @@ def __init__(
             top_p=1,
             request_timeout=60,
     ):
+        # TODO: modify this to allow Anthropic, Claude, etc
         if 'gpt' not in model_name:
```

Comment:
If more types of LLMs are supported, we probably need to generalize this line of code